### DISCUSSION TITLE
Overwhelmed HA noob trying and failing to make this work...help

### DISCUSSION THREAD

**Discussion by tblomega** [sbidy/wiz_light#89] [General]
Installed HA last night, which seems to have changed sense all the guides I was following were made.
Got Hacs installed after some to-do with it's setup being different and got my sonoff stuff going.
Now I am focusing on my Phillups/WIZ enabled bulbs.

I managed to add this github as a custom repo in HACS, I installed this project and I added
`  
-light:
  - platform: wiz_light
    name: Nightstand L
    host: 192.168.3.19
  - platform: wiz_light
    name: Nightstand R
    host: 192.168.3.20

` 
to the configuration.yaml, but for all of that there is no evidence I did anything.

I ssh'd in and tried the mv /config/custom_components/.... command listed on this git's front page (it failed as expected, as installed with HACs and not manualy but it was a shot in the dark).

I have no idea why my bulbs won't show up, so um... as a noob a one half day into Home assistant um...Help!?

**Comment by ChrisLizon** [comment]
Did you verify the structure of your configuration.yaml? (Configuration > Server Controls > Click Check Config button) 

One thing jumping out to me is the names, I believe they should be formatted with quotation marks. It's also recommended that you use underscores in your device names not spaces. I believe that Home Assistant automatically correct this, but it might be worth a try.

**Comment by jcasarini** [comment]
Hi, 

can you paste your configuration as is in configuration.yaml?

From what you already pasted, I can see there's a "-" before "lights", that's not the correct syntax.

Should be something like this:


```
light:
  - platform: wiz_light
    name: "Nightstand L"
    host: 192.168.3.19
  - platform: wiz_light
    name: "Nightstand R"
    host: 192.168.3.19
```


After that, go to Server Controls, clic Check Config.

If it's valid, restart HomeAssistant.

After the reboot go to entities and search for your lights, you won't find them in devices.

The configuration.yaml file is quite tricky to edit, needs the correct indentation, syntax. You could also try adding them from the UI in Integrations.


Hope this works, if not let me know.

Joaquin
